tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",
+        "noImplicitAny": false,
+        "sourceMap": true,
```

Comment:
Nice find!  I updated the `.npmignore` to make sure we drop `*.ts` and `*.js.map` files.  I checked the `npm pack` output, and it looks good.